### DISCUSSION TITLE
Atiesh

### DISCUSSION THREAD

**Discussion by JipiHorrible** [chromiecraft/chromiecraft#3279] [Suggestions]
It's the only obtainable legendary caster weapon during all the progression, while the melee classes have 2. It's really hard to obtain,  and the quest to do it is really fun.  Idk how hard would it be to add the splinters to the bosses loot table, and the subsequent quest. Or maybe add the old Naxx 40 as a separate instance.

**Comment by finurlig** [comment]
I like this suggestion because I'm a druid who would love to obtain the weapon, among others. I think Chromiecraft should implement a way to finish the quest at lvl 60, not necessarily by adding Vanilla Naxxramas, but in other creative ways that we know they are capable of. One alternative would be adding the splinters to AQ40 or another instance. Not only is it the only legendary caster weapon like Jipi said, but it also affects gameplay at lvl 70, with the Karazhan portal.

**Comment by Never6** [comment]
Personally i think we should stay as far away from adding naxx items as possible sure they would awesome. 
but these items and t3 are bis until 70. And with wrath talents we could probably do t4 in t3 unless the raids were overtuned.
